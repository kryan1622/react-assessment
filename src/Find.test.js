import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Find from './Components/find.js';


it('render find', () => {
  const wrapper = renderer.create(<Find/>).toJSON()
  expect(wrapper).toMatchSnapshot();
})