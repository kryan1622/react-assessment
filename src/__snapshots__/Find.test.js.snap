// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render find 1`] = `
<div>
  <br />
  <br />
  <h1>
     All Users 
  </h1>
  <br />
  <br />
  <br />
  <h4>
    Usernames
  </h4>
  <table
    className="table"
  >
    <tbody />
  </table>
</div>
`;
